src/resolution_automatic.jl

```diff
@@ -116,13 +118,33 @@ function checksum_pass(hash, fetched_path)
     true
 end
 
-function accept_terms(dd::DataDep, localpath, remotepath)
+##############################
+# Term acceptance checking
+
+"""
+    accept_terms(dd, localpath, remotepath, i_accept_the_terms_of_use)
```

Comment:
`dd` in the docstring doesn't seem very helpful. maybe spell out `datadep` here for clarity?  I don't think it has to exactly mirror the actual code